modules/tensor_mechanics/doc/content/source/materials/StrainEnergyRateDensity.md

```diff
@@ -0,0 +1,33 @@
+# Strain Energy Rate Density
+
+!syntax description /Materials/StrainEnergyRateDensity
+
+## Description
+
+This material computes the strain energy rate density, $\dot{u}$, which is defined as the
+time rate of the strain energy density (see [`StrainEnergyDensity`](/StrainEnergyDensity.md)):
+\begin{equation}
+  \label{eqn:sed_integral_def}
+  \dot{u} = \sigma : \dot{\epsilon}
+\end{equation}
+where $\sigma$ is the stress and $\epsilon$ is the strain rate. In the
```

Comment:
I guess `n_exponent` actually comes in here, right?

Reply:
Yes. I left the formula without the exponent influence since, in theory, it's just a particular case, but explained it in a comment below.